### DISCUSSION TITLE
ThirdWeb Deployment issue

### DISCUSSION THREAD

**Discussion by Teoaks** [Cyfrin/foundry-full-course-cu#307] [Q&A] [answered]
I am having troubles deploying my contract. 
When I use the command ``npx thirdweb deploy`` in the VS Code terminal I keep getting the message:

``
CMD.EXE was started with the above path as the current directory.
UNC paths are not supported. Defaulting to Windows directory.
``

and also:

``
No contracts detected in this directory. Redirecting to the thirdweb contract explore page.
√ Open this link to deploy your contracts: https://thirdweb.com/explore
``

Doesn't matter if I change the directory it is still the same issue. I also tried installing node.js on my WSL Linux, that didn't do the trick...
Did somebody have the same issue and knows how to solve this one?

**Comment by Teoaks** [comment, marked as answer]
I actually managed to solve the issue. I was simply missing the node.js from my Linux WSL. 
To install it you can refer to this youtube video: 
``https://www.youtube.com/watch?v=R73JI8rwkKc ``. 
 After that close and open your VS Code again and proceed with the command ``npx thirdweb deploy`` . 
This time it should work properly.

**Reply by cromewar** [reply]
Thanks @Teoaks

**Reply by DukeLurkin** [reply]
Thanks for this. I had to trash my terminal and then restart VS, open new terminal,  install thirdweb again then it worked.  TY

**Reply by dezlymacauley** [reply]
@Teoaks Thanks for this!!! Just got my ThirdWeb link to deploy the contract

**Reply by yashxcode** [reply]
Thanks a lot @Teoaks , I was having trouble with this but solved it now using your answer above.

**Reply by stunspotx** [reply]
This was really helpful, thanks

**Reply by sarvagnakadiya** [reply]
I am not getting the correct contract option to deploy

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85187416/3533ed60-788e-4d4d-8118-30605d4528a3)

as you can see, I am able to deploy this using create
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/85187416/20af6abe-12df-414b-a5cb-2b4fc0fb6dad)

HELP PLEASE!!!

**Comment by Teoaks** [comment]
You are welcome ! 🤗
